common/src/main/java/com/microsoft/identity/common/internal/platform/AsymmetricAlgorithm.java

```diff
@@ -22,6 +22,8 @@
  // THE SOFTWARE.
  package com.microsoft.identity.common.internal.platform;
 
+import com.microsoft.identity.common.java.crypto.Algorithm;
```

Comment:
Stray import?

Reply:
Used by getSecureHardwareState()

Reply:
It's the base class. These are migrated to common4j and therefore now belong to a different package.